optuna/visualization/matplotlib/_timeline.py

```diff
@@ -0,0 +1,119 @@
+from typing import Any
+
+from optuna._experimental import experimental_func
+from optuna.study import Study
+from optuna.trial import TrialState
+from optuna.visualization._timeline import _get_timeline_info
+from optuna.visualization._timeline import _TimelineInfo
+from optuna.visualization.matplotlib._matplotlib_imports import _imports
+
+
+if _imports.is_successful():
+    from optuna.visualization.matplotlib._matplotlib_imports import Axes
+    from optuna.visualization.matplotlib._matplotlib_imports import dates
+    from optuna.visualization.matplotlib._matplotlib_imports import patches
+    from optuna.visualization.matplotlib._matplotlib_imports import plt
+    from optuna.visualization.matplotlib._matplotlib_imports import ticker
+
+
+class _DateFormatter_Millisecond(ticker.Formatter):
```

Comment:
I don't think this class handles the millisecond information. Could you rename this class, please?

Reply:
I removed this class. The current code only displays hours, minutes, and seconds.